helm/upgrade-operator/templates/rbac.yaml

```diff
@@ -1,4 +1,55 @@
 apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
```

Comment:
I recon this is needed in the Helm chart too so I added it here but this is untested. The role itself + binding works for me. @pipo02mix could you have a look? Is it possible to install this via Helm/App already? If not we can go with this and fixup if we find issues later.

Reply:
I am on it, but yes we can add it anyway